python/cluster_management/README.md

```diff
@@ -47,16 +47,17 @@ pytest test/test_dsql_cluster_management.py [-s]
 Test execution will take around five minutes as it waits for clusters to complete activation and deletion.
 
 ### Executing single operations
+
 Files in [src/](src/) each have a `main()` function that let you exercise single operations.
 
 ```shell
 # Check each operation for its expected environment variables
-REGION_1="us-east-1" CLUSTER_ID_1="<your cluster id>" \
+CLUSTER_1_REGION="us-east-1" CLUSTER_1_ID="<your cluster id>" \
```

Comment:
If we change the `get_cluster.py` file then this will probably need to be updated to match.